File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/MuscleTestFormViewModel.cs
//$864FD440
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class MuscleTestFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class MuscleTestFormViewModel: BaseAdditionalInfoFormViewModel
    {

        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
           var muscleTestFormService = new MuscleTestFormServiceController();
           muscleTestFormService.MuscleTestFormInternal(this, objectContext);
           return (MuscleTestForm)objectContext.GetObject(this._MuscleTestForm.ObjectID, "MuscleTestForm");
        }
    }
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/MuscleTestFormViewModel.ts
//$60F3DB1C
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { MuscleTestForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class MuscleTestFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.MuscleTestFormViewModel, Core';
    @Type(() => MuscleTestForm)
    public _MuscleTestForm: MuscleTestForm = new MuscleTestForm();
}
